tests/libs/build_api_v2_test.py

```diff
@@ -92,12 +92,21 @@ def test_build_summary_for_fips(
                 "currentUsageCovid": nyc_latest["current_hospitalized"],
                 "currentUsageTotal": nyc_latest["hospital_beds_in_use_any"],
             },
+            hsaHospitalBeds={
+                "capacity": nyc_latest.get("staffed_beds_hsa"),
+                "currentUsageCovid": nyc_latest.get("current_hospitalized_hsa"),
+                "currentUsageTotal": nyc_latest.get("hospital_beds_in_use_any_hsa"),
+            },
             icuBeds={
                 "capacity": nyc_latest["icu_beds"],
-                "totalCapacity": nyc_latest["icu_beds"],
```

Comment:
This extra field was in the test, not sure, why (it doesn't exist in the API) so I removed it